src/components/Repositories/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from 'react';
+import axios from "axios";
+import UserListing from '../UserListing';
+import RepoListing from '../RepoListing';
+
+
+const url = "https://api.github.com"
+
+const Repositories = ({ username }) => {
+
+    const [repos, setRepos] = useState();
+    const [user, setUser] = useState();
+
+    useEffect(() => {
+        const getUser = async () => {
+            try {
+                const {data} = await axios.get(`${url}/users/${username}`);
+                setUser(data);
+            } catch (error) {
+                setUser();
+                console.error(error);
+            }
+        }
+        const getRepos = async () => {
+            try {
+                const {data} = await axios.get(`${url}/users/${username}/repos`);
+                setRepos(data);
+            } catch (error) {
+                setUser();
```

Comment:
would have been nice if the error showed up on screen so the user can see what went wrong